sdks/python/apache_beam/ml/transforms/tft.py

```diff
@@ -291,6 +291,41 @@ def apply_transform(
     return output_dict
 
 
+@register_input_dtype(float)
+class ScaleToGaussian(TFTOperation):
+  def __init__(
+      self,
+      columns: List[str],
+      elementwise: bool = False,
+      name: Optional[str] = None):
+    """
+    This function applies a scaling transformation on the given columns
```

Comment:
Nit: Consider a shortened message :    `This operation scales the given input column values to an approximately normal distribution with mean 0 and variance of 1. `

Reply:
done as part of the docstring update